package.json

```diff
@@ -4,128 +4,96 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "react-app-rewired --max_old_space_size=4096 build",
+    "build": "tsc && vite build",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "react-app-rewired start",
-    "test": "react-app-rewired test --colors",
+    "start": "vite",
+    "test": "vitest",
     "lint": "eslint 'src/**/*.{ts,tsx}'",
     "lint-staged": "lint-staged --config lint-staged.js",
-    "serve-build": "yarn build && yarn dlx serve -s build -p 4200",
-    "format": "yarn run prettier --write 'src/**/*.{ts,tsx}'",
+    "preview": "vite preview",
+    "format": "prettier --write 'src/**/*.{ts,tsx}'",
     "prepare": "husky install"
   },
-  "jest": {
-    "moduleNameMapper": {
-      "@aws-amplify/ui-react/styles.css": "@aws-amplify/ui-react/dist/styles.css"
-    }
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest",
-      "prettier"
-    ],
-    "plugins": [
-      "prettier"
-    ],
-    "rules": {
-      "prettier/prettier": "error"
-    }
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "packageManager": "yarn@3.3.0",
+  "packageManager": "yarn@4.0.1",
   "engines": {
-    "node": "18.16.1"
+    "node": "20.9.0"
```

Comment:
As mentioned in Discord chat, let's keep this on v18 for now.

Reply:
fixed